### PR TITLE
feat: init update draft

### PR DESCRIPTION
**Pull request by kopy-kat**

### CODE REVIEW

File: ERCS/erc-7484.md [hunk 1]
```diff
@@ -41,80 +41,85 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
 
-Takes three arguments: `module`, `attesters` and `threshold`.
+- Allow attesters to specify an expiry date for their attestations and revert during a check if an attestation is expired.
+- Implement a view function that allows an adapter or offchain client to read the data for a specific attestation.
 
-Note: `threshold` may be 0.
+#### `check` functions
 
 - The Registry MUST revert if the number of `attesters` that have made an attestation on the `module` is smaller than the `threshold`.
 - The Registry MUST revert if any `attester` has revoked their attestation on the `module`.
```
Comment (kopy-kat) [inline ERCS/erc-7484.md:109]: this is still up to date right @zeroknots

File: ERCS/erc-7484.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ This proposal standardises the interface and functionality of Module Registries,
 
 ## Motivation
 
-[ERC-4337](./eip-4337.md) standardises the execution flow of contract accounts and [ERC-7579](./eip-7579) aims to standardise the modular implementation of these accounts, allowing any developer to build modules for these modular accounts (hereafter Smart Accounts). However, adding third-party modules into Smart Accounts unchecked opens up a wide range of attack vectors.
+[ERC-4337](./eip-4337.md) standardises the execution flow of contract accounts and [ERC-7579](./eip-7579.md) standardises the modular implementation of these accounts, allowing any developer to build modules for these modular accounts (hereafter Smart Accounts). However, adding third-party modules into Smart Accounts unchecked opens up a wide range of attack vectors.
```
Comment (zeroknots) [inline ERCS/erc-7484.md]: should we use US.en?

File: ERCS/erc-7484.md (hunk 1 above)
Comment (zeroknots) [reply ERCS/erc-7484.md:109]: correct

**Review by zeroknots — APPROVED** [review]
looks good

**Pull request merged**